### ISSUE TITLE
Rerun evaluation run (exp15)

### ISSUE THREAD

**Issue by proycon** [knaw-huc/golden-agents-htr#27]
After the recent changes (#21 #22 #25 #26), a new evaluation run (exp15) is needed.

Adaptions to `process-evaluation-results.py` may be needed for this.

**Comment by proycon** [comment]
Experiment finished (see above commit), some further output inspection may be desired before we start the final production run

**Comment by proycon** [comment]
(This experiment is being rerun several times as changes to #22 concerning the final web annotation output are made)

**Comment by proycon** [comment]
Though the final run is already running, I'd still like to rerun an evaluation run as well, but for that the evaluation scripts need to be adapted because of all the recent changes.

**Comment by proycon** [comment]
I adapted the evaluation script to not break anymore on the various changes we made lately and output some results. Although the script now runs, it evaluates on a fairly shallow level and there are numerous problems. The [scores in the evaluation](https://github.com/knaw-huc/golden-agents-htr/blob/master/experiments/evaluation.exp16.log) have therefore dropped considerably compared to earlier runs.

* The higher-order 'observations' for persons, objects, locations etc are *not* evaluated as-such as the reference data operates on a lower-level. 
* The evaluation script evaluates firstname/familyname separately again, it does not evaluate the larger 'person' observation that the emerges from the context rules. The reference data is often inconsistent and uses all three.
* There seems to be quite a lot that was missed by the annotators in the reference data, for some of the auxiliary categories (material, property) it's not so important, but for objects and locations, it is.
* The evaluation script is strict about case sensitivity
* The system output often gets the wrong morphological form, the evaluation script is strict about this.
* There were quite some category changes in boedeltermen.csv, the impact on the evaluation is not clear
* The system and/or reference may have multiple output categories (=ambiguitity)
